readme.md

```diff
@@ -91,25 +95,28 @@ Once installation is working, try creating and requesting resources. Note that t
 * **Question:** Describe what the above code is adding to your schema.
 
 * **Your Answer:**
-
+The above is adding a timestamp to the schema on each record for when it is created and when it is updated.
 ---
 
 - [ ] Import the model into your `routes/series.js` file. Use either basic promises or `async/await` to update the `GET /` method to retrieve from the database instead of the given array. [This page](https://mongoosejs.com/docs/documents.html) or [this page](https://mongoosejs.com/docs/api.html#Model) may be useful.
 
 * **Question:** What method do you use to return all documents from a collection?
 
 * **Your Answer:**
-
----
+  module.exports = mongoose.model('Series', schema) will return all documents.
+  
+  ---
 
 - [ ] Update the API so that you can create documents in the database. You may either use the `<document>.save()` method or the `Model.create()` method.
 
 * **Question:** What happens if you do not include all the fields as specified in the schema?
 
 * **Your Answer:**
-
+It will fail on the schema validation.
```

Comment:
It actually won't, unless you set up validations as part of it! You can submit partial data with Mongo.